src/ui/components/VPNToggle.qml

```diff
@@ -286,7 +286,6 @@ VPNButtonBase {
         anchors.fill: toggle
         hoverEnabled: (VPNController.state === VPNController.StateOn || VPNController.state === VPNController.StateOff)
         cursorShape: Qt.PointingHandCursor
-        onClicked: clicked()
```

Comment:
@yzen will not be happy about it. He just added this line :) 
See: https://github.com/mozilla-mobile/mozilla-vpn-client/commit/eeeabb3b5cd672c85b2471a647c9361725baa96e

Reply:
@yzen Do we need this line? I'm able to connect/disconnect via keyboard without it.

Reply:
Catching up on this, looks like things are accessible with the Toggle, in onboarding (I tried with keyboard and VO on mac) 🎉